src/ListadoDeEjercicios.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import PresentacionDeEjercicio from "./PresentacionDeEjercicio";
+
+class ListadoDeEjercicios extends Component {
+
+    constructor(props) {
+       super(props);
+       this.state = {
+           ejercicios: this.props.ejercicios.todos()
+       }
+    }
+
+    propiedades(ejercicio) {
+      return {
+        component: ejercicio.component,
+        props: ejercicio.props,
+        titulo: ejercicio.titulo,
+        descripcion: ejercicio.descripcion,
+      }
+    }
+
+    render() {
+        return (
+            <div className="listado-de-ejercicios">
+                {
+                    this.state.ejercicios.map((ejercicio) => {
+                        return (
+                            <PresentacionDeEjercicio
+                                key={`presentacion-de-ejercicio-${ejercicio.props.id}`}
+                                ejercicio={ this.propiedades(ejercicio) }
```

Comment:
Otro comentario acá, estaría muy mal pasarle el ejercicio directamente?
`this.propiedades` mapea uno a uno algunos de los atributos del ejercicio